career_village.md

```diff
@@ -29,17 +27,17 @@ __Upcoming Sessions__
 
 **September 4 – 6:00 PM ET**  
 **Common Misconceptions on Starting in Cybersecurity & How You Can Stand Out**  
-*Speaker: Levi Cook | 45 min*  
+*Speaker: Levi Cook *  
```

Comment:
Extra space at the end of Levi's name is preventing proper formatting. Need to remove it for the name to be italicized on the site.